cloudnatix/README.md

```diff
@@ -0,0 +1,53 @@
+# Agent Check: CloudNatix
+
+## Overview
+
+This check provides the data from [CloudNatix][1].
+
+## Setup
+
+### Installation
+
+To install the CloudNatix check on your cluster:
+
+1. [Install CloudNatix](https://docs.cloudnatix.com/docs/tutorial) into your cluster.
+
+2. [Download the Datadog Agent][2].
+
```

Comment:
This should include a step by step on how to install the integration

Reply:
This is exactly the everything what the user needs. 

Installing CloudNatix means adding our agent into the kubernetes cluster, which has the auto-discovery properties of the integration.  Then install the Datadog agent to the same kubernetes cluster.  It will detect the auto-discovery, and then start sending the reports.  There's no step 3.

What else should I add?